package-lock.json

```diff
@@ -3284,6 +3284,12 @@
         "estraverse": "^4.1.1"
       }
     },
+    "esprima": {
```

Comment:
@amirch1 after `npm install` it found 2 moderate vulnerabilities, I've fixed them with `npm audit fix` as well